packages/twenty-server/src/engine/workspace-manager/workspace-sync-metadata/services/workspace-sync-function-metadata.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable, Logger } from '@nestjs/common';
+
+import { EntityManager } from 'typeorm';
+
+import { WorkspaceSyncContext } from 'src/engine/workspace-manager/workspace-sync-metadata/interfaces/workspace-sync-context.interface';
+
+import {
+  FunctionMetadataEntity,
+  FunctionSyncStatus,
+} from 'src/engine/metadata-modules/function-metadata/function-metadata.entity';
+import { CodeEngineService } from 'src/engine/core-modules/code-engine/code-engine.service';
+
+@Injectable()
+export class WorkspaceSyncFunctionMetadataService {
+  private readonly logger = new Logger(
+    WorkspaceSyncFunctionMetadataService.name,
+  );
+
+  constructor(private readonly codeEngineService: CodeEngineService) {}
+
+  async synchronize(context: WorkspaceSyncContext, manager: EntityManager) {
```

Comment:
does this make sense?

Reply:
I called it `synchronize` to be consistent with the other metadata synchronization method. `build` would be a more accurate name indeed